CONTRIBUTING.md

```diff
@@ -14,7 +14,8 @@
     - [Development Procedure](#development-procedure)
     - [Pull Merge Procedure](#pull-merge-procedure)
     - [Release Procedure](#release-procedure)
-    - [Point Release Procedure](#point-release-procedure)
+    - [Stable Release Update Procedure](#stable-release-update-procedure)
+    - [Stable Release Updates](#stable-release-updates)
```

Comment:
This seems inconsistent with the recent updates calling it "point release" again.

@alessio can you update this file accordingly?

Reply:
Fixed. Will double check URLs sanity shortly.